core/loadpoint.go

```diff
@@ -111,6 +116,8 @@ func NewLoadPointFromConfig(log *util.Logger, cp configProvider, other map[strin
 	}
 	if lp.VehicleRef != "" {
 		lp.vehicle = cp.Vehicle(lp.VehicleRef)
+		lp.energyPerSocStep = float64(lp.vehicle.Capacity()) * 1e3 / 100
```

Comment:
Wäre schöner das erst bei Bedarf zu setzen und nicht irgendwo in der globalen Initialisierung

Reply:
Vorschlag?